package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-request-editor",
-  "version": "1.11.0",
+  "version": "1.12.0",
```

Comment:
Need to update `package-lock.json` with this version

Reply:
updated to 1.12.0